site2/docs/reference-pulsar-admin.md

```diff
@@ -2230,20 +2230,28 @@ Options
 |`-l`, `--ledgerId`|The ledger id |0|
 |`-e`, `--entryId`|The entry id |0|
 
-### `get-deduplication`
-Get a deduplication policy for a topic.
+### `enable-deduplication`
+Enable a deduplication policy for a topic.
```

Comment:
@what's the differences between `enable-deduplication` and `get-deduplication-enabled`?

Reply:
@freeznet I am not sure. Can you take a look at this question. Thank you.